web/shared/tools/debugger/compat.tsx

```diff
@@ -0,0 +1,112 @@
+import { useCallback } from 'preact/hooks'
+
+declare module 'preact' {
+    namespace JSX {
+        interface IntrinsicElements {
+            'center': JSX.HTMLAttributes<HTMLElement>
+            'data-channel': JSX.HTMLAttributes<HTMLElement>;
+            'debug-player': JSX.HTMLAttributes<HTMLElement>;
+            'video-size-select': JSX.HTMLAttributes<HTMLElement>;
+        }
+    }
+}
+
+declare global {
+    interface Window {
+        refreshDevice(): Promise<void>;
+        startWhip(): Promise<void>;
+        startWhep(): Promise<void>;
+    }
+}
+
+export default function DebuggerCompat() {
+    const refreshDevice = useCallback(() => window.refreshDevice(), [])
+    const startWhip = useCallback(() => window.startWhip(), [])
+    const startWhep = useCallback(() => window.startWhep(), [])
+
+    return (
+        <>
+            <fieldset>
+                <legend>Common</legend>
+                <section style="display: flex;justify-content: space-evenly;flex-wrap: wrap;">
+                    <div>Stream ID: <input id="id" type="text" /></div>
+                    <div>Bearer Token: <input id="token" type="text" /></div>
```

Comment:
There Stream ID and Bearer Token URLSearchParams don't sync

if you use `debugger.html?id=777?token=xxx`, It's don't working